cmd/rekor-cli/app/pflags.go

```diff
@@ -190,6 +191,53 @@ func validateFileOrURL(v string) error {
 	return valGen().Set(v)
 }
 
+// validateFullID ensures the FullID is composed of a valid shardID, separator character, and UUID
+func validateFullID(v string) error {
+	if len(v) == sharding.FullIDLen {
+
+		// validate separator
+		if string(v[sharding.ShardIDLen]) != sharding.FullIDSeparator {
+			return fmt.Errorf("unexpected separator char in fullID: %v", v[sharding.ShardIDLen])
+		}
+
+		// validate the ShardID
+		shardID := v[:sharding.ShardIDLen]
+		shardIDTag := "required," + "len=" + fmt.Sprintf("%v", sharding.ShardIDLen) + ",number"
```

Comment:
nit: is there a reason this is not `shardIDTag := "required, len=" + fmt.Sprintf("%v",`?

Reply:
@jeffvance No, there isn't. : ) Thanks for the catch, I will update this.